packages/commonwealth/server/routes/updateCommunityCategory.ts

```diff
@@ -38,6 +44,45 @@ const updateCommunityCategory = async (
     );
   });
 
+  const { tag_ids } = req.body;
+  if (tag_ids) {
```

Comment:
If we use the optional chaining operator in tag_ids.length, like `tag_ids?.length > 0`, we can remove this if statement

Reply:
Done [here](https://github.com/hicommonwealth/commonwealth/pull/7824/commits/b5a4f0c5805f8cdb89f350c6d1d7670e04789712)